src/lib/mocha/mocha.js

```diff
@@ -70,7 +70,7 @@ Mocha.prototype.start = function (callback) {
     mochaConfig: JSON.stringify(this.options.mochaConfig)
   });
 
-  self.child = cp.fork(path.join(__dirname, 'mocha-wrapper.js'), _.union(
+  self.child = cp.fork(path.join(__dirname, 'mocha-wrapper-instance.js'), _.union(
```

Comment:
this looked like the only place dependent on the old `mocha-wrapper.js` but let me know if I missed somewhere else